### DISCUSSION TITLE
unable to run tests on the combined module and simulations input files do not run!

### DISCUSSION THREAD

**Discussion by Ali-toghraee** [idaholab/moose#17292] [Q&A Getting Started] [answered]
Hello all,
I am trying to install the new version of MOOSE with conda. The tests did not run at first but I did the following and the moose tests passed.

conda deactivate
module purge
conda activate moose
cd /path to my moose folder/
git clean -xfd # warning... this will delete any unsaved commits if you have any
cd test
make -j #my cpu core numbers
./run_tests

Now, to run my simulations, I made the executable in combined module folder(combined-opt). However the tests for that did not run leaving errors as below:

(base) atoghraee@ME-DESKTOP102: ~/ projects02/moose/modules/combined$ conda activate moose
(moose) atoghraee@ME-DESKTOP102: ~/ projects02 / moose/ modules/ combined$ ./ run_tests -j 90
Building and linking hit...
failed to import pyhit - try running "make hit" in the $MOOSE_DIR/test directory.
(moose) atoghraee@ME-DESKTOP102:~/ projects02/ moose/modules / combined$ 

 
**I should mention that I already have a moose from 2019 in the projects folder, So to install the new moose I just created another folder under projects02**

**Comment by GiudGiud** [comment]
Hello

It looks like you are missing pyhit. Could you please install it with conda install moose-pyhit

Guillaume

**Reply by aeslaughter** [reply]
I don't think we have a moose-pyhit package in conda.

**Comment by aeslaughter** [comment, marked as answer]
You might be running into a PYTHONPATH problem, make sure you don't have it set to something that points it to the projects/moose location. Also, make sure you don't have $MOOSE_DIR set incorrectly for where you are working.

**Reply by aeslaughter** [reply]
Also make sure you have the conda environment loaded.
